CMakeLists.txt

```diff
@@ -260,7 +260,9 @@ if(PROJECT_SOURCE_DIR STREQUAL PROJECT_BINARY_DIR)
 endif()
 
 if (NOT CMAKE_BUILD_TYPE)
-    set( CMAKE_BUILD_TYPE Release)
+   set( CMAKE_BUILD_TYPE Release)
+elseif(CMAKE_BUILD_TYPE STREQUAL "PureDebug")
```

Comment:
What about adding this: http://public.kitware.com/pipermail/cmake/2014-June/057852.html 

perhaps with a call to the uppercase as the author suggests?